apps/events-helsinki/i18nRoutes.config.js

```diff
@@ -20,13 +20,28 @@ const i18nRoutes = {
     { source: '/sidor/:slug*', locale: 'sv' },
   ],
   '/accessibility': [
-    { source: '/saavutetavuusseloste', locale: 'fi' },
-    { source: '/tillgäntglighetsutlåtande', locale: 'sv' },
```

Comment:
I don't think that these are in any use at all, so they could both be removed. Same thing in `/about` -path. These pages should be served via CMS pages and they should be under `/pages` -root, which is a root for dynamic pages.

Example: https://tapahtumat.test.hel.ninja/fi/sivut/saavutettavuusseloste

Reply:
these can be removed from every monorepo app!

Reply:
Removed in 8de789f1ef34ec53de94241e3c6717a413e761f2